File: app/src/main/java/com/lkms/data/model/java/Team.java
package com.lkms.data.model.java;

import com.google.gson.annotations.SerializedName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * Đại diện cho một hàng trong bảng "Team".
 * Bảng này có khóa chính phức hợp (experimentId, userId).
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Team {

    // Khóa ngoại đến Experiment
    @SerializedName("experimentId")
    private Integer experimentId;

    // Khóa ngoại đến User
    @SerializedName("userId")
    private Integer userId;

    @SerializedName("status")
    private String status;
}

File: app/src/main/java/com/lkms/data/model/Team.kt
package com.lkms.data.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

/**
 * Đại diện cho một hàng trong bảng "Team".
 * Bảng này có khóa chính phức hợp (experimentId, userId).
 */
@Serializable
data class Team(
    // Khóa ngoại đến Experiment
    @SerialName("experimentId")
    val experimentId: Int,

    // Khóa ngoại đến User
    @SerialName("userId")
    val userId: Int,

    @SerialName("status")
    val status: String? = null
)